tests/Integration/Subscriber/CDN/CDNSubscriber/TestRewrite.php

```diff
@@ -64,38 +64,6 @@ public function testShouldReturnOriginalWhenCDNDisabled() {
         );
     }
 
-    public function testShouldReturnOriginalWhenDONOTROCKETOPTIMIZE() {
```

Comment:
Why this test was removed? I think this one needs to be fixed with mocking the constants and get rid of @runInSeparateProcess

Reply:
It was completely blocking the execution of integration tests. At the time we didn't have the alternative for the constants, and I needed the tests to run. It will be added back by the work on the PR #2231